deploy/helm/kubecf/assets/operations/sequencing.yaml

```diff
@@ -0,0 +1,139 @@
+# We specify the new dependencies (I) and remove the old serialization
+# hints (II).
+#
+# I. Custom pod dependencies for startup sequencing
+##
+# ATTENTION: Annotations are strings. While the cf-operator's
+# controller will unmarshall the value as a json array later on,
+# specification must be done as a string using json syntax inside.
+##
+# Short form of the ops changes below.
+# Dependencies pointing to dependents.
+# Order here matches order of ops instructions.
+##
+# auctioneer <-- diego-api		(diego only, not eirini)
+# scheduler  <-- api
+# cc-worker  <-- api
+# router     <-- routing-api <-- uaa
+#                tcp-router  <-- uaa
+#                log-cache   <-- uaa
+#                credhub     <-- uaa
+#                diego-cell  <-- uaa	(diego only, not eirini)
+#
+# asactors  <- asdatabase
+# asapi     <- asdatabase
+# asmetrics <- asdatabase
+# asnozzle  <- asdatabase
+##
+
+{{- if not .Values.features.eirini.enabled }}
+
+- type: replace
+  path: /instance_groups/name=auctioneer/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "diego-api" ]'
+
+{{- end }}
+
+- type: replace
+  path: /instance_groups/name=scheduler/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "api" ]'
+- type: replace
+  path: /instance_groups/name=cc-worker/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "api" ]'
+
+- type: replace
+  path: /instance_groups/name=router/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "routing-api" ]'
+
+- type: replace
+  path: /instance_groups/name=routing-api/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "uaa" ]'
+- type: replace
+  path: /instance_groups/name=tcp-router/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "uaa" ]'
+- type: replace
+  path: /instance_groups/name=log-cache/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
+  value: '[ "uaa" ]'
+- type: replace
+  path: /instance_groups/name=credhub/env?/bosh/agent/settings/annotations/quarks.cloudfoundry.org~1wait-for
```

Comment:
I believe credhub is optional too.

Reply:
Checking ...

Reply:
Indeed. Just enabled by default.

Reply:
Fixed with commit 1233db79137d40221c8bd147de8e8ffbeb67a4f3